griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -64,6 +75,8 @@ class OpenAiChatPromptDriver(BasePromptDriver):
         ),
         kw_only=True,
     )
+    tool_choice: str = field(default="auto", kw_only=True, metadata={"serializable": False})
```

Comment:
There's no doc comment for this. What other values would you supply other than `"auto"`?